src/stories/material-link/material-link.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export interface MaterialLinkList {
+  url: string;
+  text: string;
+}
+
+export interface MaterialLinkProps {
+  title: string;
+  subTitle?: string;
+  linkList: MaterialLinkList[];
+}
+
+const MaterialLink: React.FC<MaterialLinkProps> = ({
```

Comment:
I think a more proper name for this would be something like:
`Series` or `SeriesLine`
It is an information about the series data on the shown material.